src/elements/content-preview/ContentPreview.js

```diff
@@ -339,6 +339,8 @@ class ContentPreview extends React.PureComponent<Props, State> {
             this.loadPreview();
         } else if (hasTokenChanged) {
             this.updatePreviewToken();
+            this.api.updateToken(token);
+            this.fetchFile(currentFileId);
```

Comment:
Is this required?

Reply:
It is in the case if you're changing the key and the filename. There is another strategy I can employ to refresh the component in the story.

Reply:
I removed the call, and added the refresh button to the story.